apps/wallet-dashboard/components/dialogs/unstake/views/UnstakeView.tsx

```diff
@@ -156,14 +156,24 @@ export function UnstakeView({
             </DialogLayoutBody>
 
             <DialogLayoutFooter>
+                {(!isUnstakeTxPending || isTransactionPending || !delegationId) && (
```

Comment:
how come this is the condition to show not having enough gas? im not following the logic here

Reply:
shouldnt it be based on an error message that matches the not enough balance error? i think it would be more robust like that, this condition to show not enough gas seems a bit arbitrary